azure-pipelines.yml

```diff
@@ -5,14 +5,16 @@ pool:
   vmImage: 'windows-latest'
 
 steps:
+- checkout: self
+  submodules: recursive
+  
 - task: BatchScript@1
   displayName: 'Run VsDevCmd.bat'
   inputs:
     filename: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat'
     modifyEnvironment: true
 
-- task: PowerShell@2
-  displayName: Build and Test
+- task: BatchScript@1
+  displayName: 'Run BuildAndTest.cmd'
   inputs:
-    targetType: filePath
-    filePath: ./scripts/BuildAndTest.ps1
+    filename: 'BuildAndTest.cmd'
```

Comment:
>cmd [](start = 28, length = 3)

Just curious why you chose not to invoke the PowerShell script directly?

Reply:
i thought that was the issue. i will revert and test again :)

---
In reply to: [534552440](https://github.com/microsoft/sarif-visualstudio-extension/pull/301#discussion_r534552440) [](ancestors = 534552440)